extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/test/java/io/quarkus/resteasy/reactive/server/test/resource/basic/resource/MatchedResource.java

```diff
@@ -60,7 +60,7 @@ public String start() {
         return "started";
     }
 
-    @Path("start")
+    @Path("startXML")
```

Comment:
Wait a minute, this is a false positive, isn't it? This endpoint consumes XML, while the other will be selected for non-XML since it's less specific.